resources/lib/ui/control.py

```diff
@@ -55,7 +55,10 @@ def xbmc_add_player_item(name, url, iconimage=''):
     liz.setProperty("fanart_image", __settings__.getAddonInfo('path') + "/fanart.jpg")
     liz.setProperty("Video", "true")
     liz.setProperty("IsPlayable", "true")
-    liz.addContextMenuItems([], replaceItems=False)
+    #cmd = 'XBMC.Notification({},{})'.format("9anime",url.replace("playlink&url=/",""))
```

Comment:
please cleanup,
also, instead of replacing playlink with download, we should add`&download=1` to download url `&download=0` for original url

then in playback route parse `download` flag value and either playitem or call aria2_download

Reply:
Good suggestion.